.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  release-please:
+    if: "${{ !contains(github.event.pull_request.title || github.event.issue.title, 'chore(main): release') }}"
```

Comment:
@o-leolleo 
release.yml workflow updated to ignore self-created PRs. 
Seems that's it for this repo regarding unit tests for now, unless I'm missing something.. as I'm not seeing any tests setup. 
Let me know, thanks

Reply:
resolved in priv mgs